docs/master/api-reference/directives.md

```diff
@@ -476,6 +476,10 @@ directive @can(
   Additional arguments that are passed to `Gate::check`. 
   """
   args: [String!]
+  """
+  Pass input variables as arguments that are passed to `Gate::check`. 
+  """
+  input: Boolean!
```

Comment:
Can we find a name for this that is more clear? How about `injectArgs`?

Reply:
Yeah, I had the same thought. +1 for `injectArgs`